### ISSUE TITLE
add your name to the CREDITS.md

### ISSUE THREAD

**Issue by domsbytes** [wwcodecolorado/scrollodex#16]
once its created please add your full name to this markdown and link it to your github profile

**Comment by BayLadyCoder** [comment]
I'm adding this file (CREDITS.md)

**Comment by CinnamonXI** [comment]
is this still open?

**Comment by jischr** [comment]
@CinnamonXI You must complete another PR in this repo before we can add you to the contributors. Just clarified in the issue description so its clear.
